aws/microservice-ecommerce/blueprint.yaml

```diff
@@ -63,6 +63,10 @@ parameters:
   type: Input
   description: Store Admin password
   value: admin
+- name: XLDUrlForXLR
```

Comment:
why is this needed? we were actually getting rid of such stuff

Reply:
XLD and XLR are inside the docker cluster, there was an XLR step failing because it was trying to access XLD using 'localhost'